docs/multi-organization-guide.md

```diff
@@ -0,0 +1,182 @@
+
+The following considerations should be observed when creating Deployment Maps for an multi-organization ADF setup:
+1. Create Organization-agnostic deployment maps
+    - As a best-practice, deployment maps should be free of any hardocded AWS Account Ids for Deployment Map Targets, unless the deployment is destined for a single AWS Organization.
+    - Instead, target AWS accounts via `Account Names`, `Account Tags` or `OU Paths`
+    which ADF will then use to dynamically generate the respective Account IDs 
+    Target List when updating the Pipelines. 
+2. Consider AWS service limits for AWS CodePipeline
+    - In a large enteprise setup the number of targets in a "prod" AWS Organizations AWS CodePipeline
+  stage may be much greater than in predecessing stages of the "int" and "prod" AWS Organizations.
+    - Review the Codepipeline action limitations.
+    - ADF distributes targets across AWS CodePipeline stages within a AWS CodePipeline on a best efforts basis, however 
+    deployments may need to be distributed across multiple AWS CodePipeline when upper limits are reached.
+    - The current limits are ([AWS CodePipeline Limits](https://docs.aws.amazon.com/codepipeline/latest/userguide/limits.html))
+      - 1000 AWS CodePipeline per AWS Account per region
+      - 500 Actions per AWS CodePipeline
+      - 50 Actions per AWS CodePipeline Stage
+    - This implies that a single ADF pipeline can target 500 AWS accounts max. At this point latest, the ADF pipeline needs to be manually rebalanced into smaller individual ADF pipelines.
+3. Allow for empty deployment map targets
+    - With the adfconfig setting `allow-empty-target` ([ADF Admin Guide](admin-guide.md)), ADF can be instructed to ignore any target that is not resolvable or empty (because no AWS Accounts exists in it). It is suggested to set this setting to `True`. Eventhough the OU structure and general setup across the different AWS Organization stages is usually identical, the number of created AWS Accounts might not. With this setting to `True`, temporary empty OUs are just ignored and do not lead to an error.
+
+ 4. The source branch for the application code may be different per AWS Organization
+    - The above described custom `adfconfig` configuration allows a different default 
+  branch to be specified in the path `config.scm.default-scm-branch` per AWS Organization
+
+### 3. Make the AWS Organization Stage Context Available in Codepipeline Build jobs
+ADF applications often contain environment / AWS Organization stage specific configuration files. 
+In order to allow AWS Codebuild to select the proper configuration context for an application, the environment / AWS Organization stage context needs to be made available. 
+A simple pattern to solve this problem is the introduction of the SSM parameter `adf/org/stage` in  the buildspec file of the application.
+The following snippet shows the header of such a `codebuild.yaml` file. 
+```
```

Comment:
Can you add the `yaml` keyword after these three backticks?
Plus add an empty line in front and after the code block?

Reply:
done